torch/_inductor/config.py

```diff
@@ -138,6 +138,8 @@
 # fuse even in cases without common reads
 aggressive_fusion = False
 
+debug_fusion = os.environ.get("TORCHINDUCTOR_DEBUG_FUSION") == "1"
```

Comment:
comment explaining what this does, or perhaps remove it.

Reply:
I'll comment since I still want this to show what nodes get fused together for each kernel. Would be usefully for debug fusion.